src/libs/PolicyUtils.ts

```diff
@@ -466,6 +471,21 @@ function getXeroBankAccountsWithDefaultSelect(policy: Policy | undefined, select
     }));
 }
 
+function getPoliciesConnectedToSageIntacct(policies: OnyxCollection<Policy>): Policy[] {
+    return Object.values(policies ?? {}).filter<Policy>((policy): policy is Policy => !!policy && !!policy?.connections?.intacct);
```

Comment:
If we're only using to check the length then why do we need to filter. Can we just use `.some` ?

Reply:
You can ignore this comment as I can see we're looping through the policies in one of the pages.